actors/builtin/multisig/multisig_test.go

```diff
@@ -186,6 +185,8 @@ func TestApprove(t *testing.T) {
 	const noUnlockDuration = int64(0)
 	const numApprovals = int64(2)
 	const txnID = int64(0)
+	const fakeMethod = abi.MethodNum(42)
+	var fakeParams = abi.MethodParams{1, 2, 3, 4, 5}
```

Comment:
In a follow-up, please delete the `abi.MethodParams` type and just use []byte. It's legacy.